Source/Controllers/DataControllers/SPTableData.m

```diff
@@ -498,9 +509,23 @@ - (NSDictionary *) informationForTable:(NSString *)tableName fromDatabase:(NSStr
     NSArray *syntaxResult = [theResult getRowAsArray];
     NSArray *resultFieldNames = [theResult fieldNames];
 
+    SPLog(@"syntaxResult: %@", syntaxResult);
+    SPLog(@"resultFieldNames: %@", resultFieldNames);
+
+    // MARK: removed isMySQL8 == YES, so this is used for all versions
+    if([database isEqualToString:SPMySQLInformationSchemaDatabase]){
```

Comment:
Is this the best check we should use to determine when to use the view logic? Should be perhaps check the structure of the response instead?